asv.conf.json

```diff
@@ -76,8 +76,8 @@
     // followed by the pip installed packages).
     //
     "matrix": {
-        "fakeredis": [""],
-        "pytest": [""],
+        "pip+fakeredis": [""],
+        "pip+pytest": [""],
```

Comment:
Notes: I've confirmed that adding `pip+` is required in this configuration. Without it, the following error occurs:
>    ModuleNotFoundError: No module named 'fakeredis'